pkg/target/http.go

```diff
@@ -550,6 +557,43 @@ func (ht *HTTPTarget) retrieveHeaders(msg *models.Message) map[string]string {
 	return msg.HTTPHeaders
 }
 
+func (ht *HTTPTarget) gimmeBody(group []*models.Message) (templated []byte, success []*models.Message, invalid []*models.Message) {
+	if ht.debugMode {
+		if len(group) > 1 {
+			panic("Debug mode is not supported with batching")
```

Comment:
Maybe we should support batching for the non-POC implementation - we don't want the downstream tooling to depend on a behaviour we fully intend to change. 🤔